libsql/src/database/builder.rs

```diff
@@ -226,6 +228,12 @@ cfg_replication! {
 
         }
 
+        pub fn namespace<T>(mut self, namespace: T) -> Builder<RemoteReplica> 
```

Comment:
Would `pub fn namespace(mut self, namespace: impl Into<String>) -> Builder<RemoteReplica>` do the same?

Reply:
it wouldn't compile afaik

Reply:
ur right, fixed